03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,48 @@
 <!DOCTYPE html>
 <html>
-    <head>
+  <head>
         <title>The Daily Planet</title>
         <link rel="stylesheet" href="css/daily-planet-style.css"/>
     </head>
     <body>
-        <!-- You code start here -->
-    </body>
-</html>
\ No newline at end of file
+      <header>
+        <img src="https://s-media-cache-ak0.pinimg.com/564x/09/3e/12/093e12c80c9402d19518408e3eba3388.jpg" alt="Image of Logo" title="Daily Planet" style="height:100px" "width:100px"/>
+        <h1>Daily Planet</h1>
+        <nav>
+          <a href="#local">Local</a>
+          <a href="#sports">Sports</a>
+          <a href="#Economy">Economy</a>
+        </nav>
+      </header>
+      <main>
+        <section>
+          <h2>Rio Olympics athletes advised to not put their heads underwater</h2>

```

Comment:
Your code looks good. It is missing one element, <article>, which you would nest just inside the <section> element. You would then nest the <h2>.. etc inside the <article>.